tests/sitemaps_tests/test_http.py

```diff
@@ -185,18 +184,6 @@ def test_sitemap_get_urls_no_site_2(self):
         with self.assertRaises(ImproperlyConfigured):
             Sitemap().get_urls()
 
-    def test_sitemap_item(self):
```

Comment:
I don't sure it is legal to move somebody's valid code from module to module without any announcement.